cli/src/semgrep/meta.py

```diff
@@ -609,6 +609,12 @@ def to_project_metadata(self) -> out.ProjectMetadata:
         res.pull_request_author_image_url = uri_opt(
             self.glom_event(T["pull_request"]["user"]["avatar_url"])
         )
+        repo_id = os.getenv("GITHUB_REPOSITORY_ID")
```

Comment:
Can/should we add some snapshot tests for ProjectMetadata? All the examples that updated only snapshot the untyped meta json by the looks of it.

Reply:
We could ... but they would be redundant with meta.json which is a superset right now.